### ISSUE TITLE
Generate Distogram and MSA coverage 

### ISSUE THREAD

**Issue by Qrouger** [KosinskiLab/AlphaPulldown#256]
Hi,
I wanted to generate Distogram and MSA coverage plot for AlphaPulldown. As feature.pkl is not present, I had to modify the AlphaFold plot script to use the result_model[...].pkl from AlphaPulldown. If you want to generate these graphics, I shared the scripts on my github page just here: https://github.com/Qrouger/Utils_APD
![result_model_1_multimer_v3_pred_0 pkl dmap](https://github.com/KosinskiLab/AlphaPulldown/assets/144003341/b47ee6c4-6236-4f42-9b88-5a8b2119cd42)
[KikA_coverage_LDDT.pdf](https://github.com/KosinskiLab/AlphaPulldown/files/14297460/KikA_coverage_LDDT.pdf)

Best regards.
Quentin Rouger.

**Comment by Qrouger** [comment]
Hi, thanks for your reply. If you were planning to make these plots in the future , I think it is not necessary for the code to be directly integrated into your pipeline. Otherwise I would like to integrate it.

Best,

Quentin Rouger.

**Comment by gchojnowski** [comment]
Hi Quentin,

we have a code generating flattened distograms in the suite already and just need to expose it. I'm wondering, do you plot average predicetd distances on the distance map? 

g

**Comment by Qrouger** [comment]
Hi @gchojnowski , Yes I think the predicted distance is an average. It generated from result_*.pkl ["distogram"]["bin_edges"] and ["distogram"]["logits"].  The distance is not made between Ca-CA or Cb-Cb.

Best,
Quentin Rouger

**Comment by jkosinski** [comment]
> Hi, thanks for your reply. If you were planning to make these plots in the future , I think it is not necessary for the code to be directly integrated into your pipeline. Otherwise I would like to integrate it.
> 
> Best,
> 
> Quentin Rouger.

OKay, and @gchojnowski, we also have have functions for MSA plotting in ColabFold style, right?

**Comment by gchojnowski** [comment]
everything is there
g

**Comment by jkosinski** [comment]
OK! I think we should expose them in scripts as well. I will create a separate feature request for that.

In any case, @DimaMolod @dingquanyu What do you think about adding a short one-sentence comment and link to https://github.com/Qrouger/Utils_APD in the analysis section of our manuals? Or in some separate section called "Integration with other tools". 

One potentially useful unique feature @Qrouger has is the export of restraints for Rosetta based on the distogram.
